### PR TITLE
AAP-7870A fixed git merge conflict in 2.3

### PR DESCRIPTION
**Pull request by emurtoug**
Addendum to [AAP-7870](https://issues.redhat.com/browse/AAP-7870), PR [717](https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/pull/717): fixed git merge conflict within upstream/2.3.

### CODE REVIEW

File: downstream/modules/platform/proc-synchronizing-rpm-repositories-by-using-reposync.adoc [hunk 1]
```diff
@@ -28,21 +28,14 @@ To perform a reposync you need a RHEL host that has access to the Internet. Afte
...
-. If you plan to use Red Hat Single Sign-On (RHSSO) sync these repositories.
-=======
 . If you plan to use Red Hat Single Sign-On (RHSSO) you must also sync these repositories.
->>>>>>> 161c95ba (Fixed formatting and spacing issues (#660))
 
 .. jb-eap-7.3-for-rhel-8-x86_64-rpms
 .. rh-sso-7.4-for-rhel-8-x86_64-rpms
 
 +
 After the reposync is completed your repositories are ready to use with a web server.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-synchronizing-rpm-repositories-by-using-reposync.adoc:38]:
```suggestion

```
Suggestion: delete these 2 lines, as the info is replicated in the next step (line 39)

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
There's a duplicated line. Otherwise, LGTM.

File: downstream/modules/platform/proc-synchronizing-rpm-repositories-by-using-reposync.adoc (hunk 1 above)
Comment (emurtoug) [reply downstream/modules/platform/proc-synchronizing-rpm-repositories-by-using-reposync.adoc]: Change made; thanks, Aine!

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**